.github/workflows/ci.yml

```diff
@@ -94,6 +103,15 @@ jobs:
       shell: bash
       run: ${{matrix.platform.prefix}} cmake --build $GITHUB_WORKSPACE/build --config ${{ matrix.type.name == 'Debug' && 'Debug' || 'Release' }} --target install
 
+    - name: Run Tests
+      if: matrix.platform.name != 'Android' && matrix.config.name != 'iOS'
+      uses: nick-fields/retry@v2
+      with:
+        max_attempts: 10
+        timeout_minutes: 3
+        shell: bash
+        command: ${{matrix.platform.prefix}} cmake --build $GITHUB_WORKSPACE/build --config ${{ matrix.type.name == 'Debug' && 'Debug' || 'Release' }} --target runtests
+
```

Comment:
Why add this when the tests already get run as part of the `install` target?

Reply:
They don't any more since `runtests` was removed from all. Tests are run by explicitly building `runtests` and don't run when building any other target. This gives us more control over the timing and environment we run tests in, such as in this case retrying just running the tests if they spuriously fail. This is also kind of a solution to the issues brought up in #2429 and #2430.

Reply:
> They don't any more since runtests was removed from all

No, not yet. https://github.com/SFML/SFML/pull/2430

Reply:
I removed it from `ALL` in this PR already.

Reply:
Ah I see now. That should let us remove the `${{matrix.platform.prefix}}` from the preceding build command.